src/commands/games/blackjack.ts

```diff
@@ -206,8 +210,19 @@ export class BlackjackCommand extends Command {
     }
 
     // check player balance and see if it can cover the bet amount
-    if (playerBalance! < bet)
-      return message.reply(`you don't have enough coins to place that bet. ${getEmojiByName('codeySad')}`);
+
+    if (playerBalance! < bet) {
+      if (betString.endsWith('%')) {
+        const percentNeeded = playerBalance / (playerBalance / 10);
+        return message.reply(
+          `you don't have enough coins to place that bet. (you'd need at least ${percentNeeded}%) ${getEmojiByName(
+            'codeySad'
+          )}`
+        );
```

Comment:
Could we cap the % at 100% instead? In other words, if the player types a value greater than 100%, then the bot will reply that the maximum bet is 100%.

Reply:
Done. See line 203.

Reply:
I see that you added this check. If I am not mistaken, this `if` block can be removed? You would only not have enough coins to place a bet if it is over 100%. In addition, it looks like `percentNeeded` always evaluates to 10, which does not look right?

Reply:
I agree, let's just remove this if block. This is not necessary in terms of code logic or user experience (i.e. I'd rather know how many more coins I need to place that bet, instead of what % more I need).

Reply:
Good point, I removed it